stdlib/concurrent/futures/process.pyi

```diff
@@ -153,7 +178,17 @@ class ProcessPoolExecutor(Executor):
     _executor_manager_thread_wakeup: _ThreadWakeup
     _result_queue: SimpleQueue[Any]
     _work_ids: Queue[Any]
-    if sys.version_info >= (3, 7):
+    if sys.version_info >= (3, 11):
+        def __init__(
```

Comment:
It's a private attribute, but we seem to be typing other private attributes for this class 🤷‍♂️

```suggestion
        _max_tasks_per_child: int | None
        def __init__(
```

Reply:
I don't think we should add this just because other private attributes are there too. Stubtest isn't good at checking instance attributes, so we won't notice if it's renamed or removed for whatever reason.

Reply:
Yeah fair, feel free to ignore. I have no strong opinion on this.

Reply:
Yeah, we should only add private attributes when someone actually needs them. There's quite a lot of private stuff in this file; not sure we really need it all.